services/retentionmgr/metadataDep.go

```diff
@@ -352,6 +394,32 @@ func (t *metadataDepImpl) DeleteConsumerGroupExtent(cgID consumerGroupID, extID
 	return nil
 }
 
+func (t *metadataDepImpl) DeleteConsumerGroup(destID destinationID, cgID consumerGroupID) error {
```

Comment:
Suggest to keep the name as the same as metadata. DeleteConsumerGroup in metadata is a different API

Reply:
done.